### PR TITLE
Allow specifying resources for BuildBuddy container

### PR DESCRIPTION
**Pull request by ashleydavies**
This adds the ability to define requests and limits for the BB deployment

### CODE REVIEW

**Comment by ashleydavies** [comment]
cc @siggisim (sorry if you already got a notification for this)

File: charts/buildbuddy/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: BuildBuddy Open Source
 name: buildbuddy
 version: 0.0.31 # Chart version
```
Comment (siggisim) [inline charts/buildbuddy/Chart.yaml:4]: Could you bump the chart version and leave the app version at 2.1.0?

**Review by siggisim — CHANGES_REQUESTED** [review]
Thanks for the bump! I'd gotten the initial notification but was eating lunch and forgot to loop back.

Just one minor change, then this should be good to go!

File: charts/buildbuddy/Chart.yaml (hunk 1 above)
Comment (ashleydavies) [reply charts/buildbuddy/Chart.yaml]: Ah, sorry! I totally blanked and just saw a semver and incremented it 😅 fixed

Comment (siggisim) [reply charts/buildbuddy/Chart.yaml]: I've been there!

**Review by siggisim — APPROVED** [review]
Lgtm, thanks for the addition!

**Comment by ashleydavies** [comment]
Thanks for landing so promptly! 😄

**Pull request merged**